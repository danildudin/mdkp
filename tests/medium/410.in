28 8 0
933 332 607 623 168 308 849 48 947 441 319 234 132 178 389 769 928 144 500 811 311 781 230 68 870 598 635 270 
270 20 473 828 298 422 192 372 923 452 163 684 863 107 144 909 778 399 783 741 56 196 390 126 196 674 806 375 
84 689 497 250 426 790 9 179 124 800 225 531 378 26 457 229 886 805 560 309 890 327 609 362 450 434 639 141 
149 969 615 758 321 615 196 992 94 516 460 163 751 698 610 704 128 991 325 804 621 882 576 201 887 534 983 27 
386 363 995 127 768 685 7 642 268 782 275 921 461 525 358 70 577 83 711 965 830 521 731 159 552 372 31 903 
70 739 384 236 1000 940 537 513 192 354 154 87 978 387 687 566 80 912 17 528 841 463 177 296 995 389 734 626 
413 297 404 804 52 195 309 778 449 456 391 866 146 239 794 295 897 190 17 868 785 460 461 344 72 550 760 133 
37 338 452 418 550 102 830 862 765 129 996 158 374 165 443 777 755 302 58 497 597 373 774 673 506 996 960 516 
271 166 165 767 380 88 466 248 580 312 222 468 837 686 600 537 43 232 77 885 49 324 813 870 520 771 671 516 
5059 8765 5783 9228 6605 9935 6226 9609 
