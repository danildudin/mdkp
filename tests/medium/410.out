8195
11
0 2 6 8 14 15 16 19 20 21 24 
